tests/e2e/parameters/infrastructure.py

```diff
@@ -3,34 +3,31 @@
 from aws_cdk import CfnOutput
 from aws_cdk import aws_appconfig as appconfig
 from aws_cdk import aws_iam as iam
-from aws_cdk import aws_ssm as ssm
 
+from tests.e2e.utils.data_builder import build_service_name
 from tests.e2e.utils.infrastructure import BaseInfrastructure
 
 
 class ParametersStack(BaseInfrastructure):
     def create_resources(self):
         functions = self.create_lambda_functions()
-        self._create_parameter_string(function=functions["ParameterStringHandler"])
         self._create_app_config(function=functions["ParameterAppconfigFreeformHandler"])
 
-    def _create_parameter_string(self, function: Function):
-        parameter = ssm.StringParameter(
-            self.stack, id="string_parameter", parameter_name="sample_string", string_value="Lambda Powertools"
-        )
-
-        parameter.grant_read(function)
-
-        CfnOutput(self.stack, "ParameterString", value=parameter.parameter_name)
-        CfnOutput(self.stack, "ParameterStringValue", value=parameter.string_value)
-
     def _create_app_config(self, function: Function):
 
-        cfn_application = appconfig.CfnApplication(self.stack, id="appconfig-app", name="appe2e", description="appe2e")
+        service_name = build_service_name()
+
+        cfn_application = appconfig.CfnApplication(
+            self.stack, id="appconfig-app", name=f"appe2e{service_name}", description="appe2e"
```

Comment:
```suggestion
            self.stack, id="appconfig-app", name=f"powertools-e2e-{service_name}", description="Lambda Powertools End-to-End testing for AppConfig"
```

making it more explicit in the event of dangling resources caused by network split